classes-recipe/image_types_qcom.bbclass

```diff
@@ -45,6 +47,7 @@ create_qcomflash_pkg() {
 
     # rootfs image
     install -m 0644 ${IMGDEPLOYDIR}/${IMAGE_LINK_NAME}.${IMAGE_QCOMFLASH_FS_TYPE} ${QCOM_ROOTFS_FILE}
+    ln -sf ${IMAGE_LINK_NAME}.${IMAGE_QCOMFLASH_FS_TYPE} ${IMGDEPLOYDIR}/${QCOM_ROOTFS_FILE}
```

Comment:
if you build several images , that would not work quite well , right? Perhaps we should customize the partition.conf file instead so that we change the default image filename there?

Reply:
right, but by choosing `partition.conf` customization path, we then would also need to generate a set of xml as well for each image leading to "overpolluted" deploy dir with:
```
core-image-base-partition.xml
core-image-base-rawprogram0.xml
...
```

I was initially thinking about a subdirectory, however on the other side, perhaps the desire to use artifacts for flashing directly from the deploy dir is not such a necessity (too much effort for the sake of it)